telos-backend/src/routes/api/dashboard.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+
+const router = express.Router();
+
+// get widgets which appear on the dashboard for a date range.
+// query params:
+// 'widget': type of widget i.e. 'all', 'calendar', etc.
```

Comment:
Should all be an option? I thought the dashboard was only for one widget at a time.